client/reader/search-stream/index.jsx

```diff
@@ -100,6 +100,9 @@ class SearchStream extends React.Component {
 		const wideDisplay = this.props.width > WIDE_DISPLAY_CUTOFF;
 		const showFollowByUrl = resemblesUrl( query );
 		const queryWithoutProtocol = withoutHttp( query );
+		const segmentedControlStyle = ! wideDisplay
+			? { bottom: 15, right: 15 }
```

Comment:
Could we have this apply a class name (perhaps `is-wide` when on a wide display) so that these values can be defined in CSS instead?

Reply:
Great catch, thanks @bluefuton I switched this up and committed the change feel free to take another test.